src/output.rs

```diff
@@ -17,10 +18,14 @@ pub enum Argument {
 }
 
 impl FilterPrototype {
-    pub fn new(name: &str, arguments: Vec<Argument>) -> FilterPrototype {
+    pub fn new(name: &str,
+               arguments: Vec<Argument>,
+               named_arguments: Vec<(String, Argument)>)
+               -> FilterPrototype {
         FilterPrototype {
             name: name.to_owned(),
             arguments: arguments,
+            named_arguments: named_arguments,
```

Comment:
> I'm not sure what you mean by "applying". I guess we'll have to change the signature of the fn for the tags, introducing a breaking change :(.

That is what I mean.  Currently, the named arguments aren't being passed to the filter.